frontend/src/metabase/public/containers/PublicOrEmbeddedQuestion/PublicOrEmbeddedQuestionView.tsx

```diff
@@ -31,7 +31,6 @@ export function PublicOrEmbeddedQuestionView({
   parameterValues,
   setParameterValue,
   setParameterValueToDefault,
-  background,
```

Comment:
Remove the ability to set `background` since static questions don't support removing background, so it makes no sense to allow setting this via prop.